src/main/kotlin/net/maizegenetics/phgv2/cli/CreateHaplotypeVCF.kt

```diff
@@ -0,0 +1,229 @@
+package net.maizegenetics.phgv2.cli
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import com.google.common.collect.Range
+import com.google.common.collect.RangeMap
+import com.google.common.collect.TreeRangeMap
+import htsjdk.samtools.SAMSequenceDictionary
+import htsjdk.samtools.SAMSequenceRecord
+import htsjdk.variant.variantcontext.VariantContext
+import htsjdk.variant.variantcontext.VariantContextBuilder
+import htsjdk.variant.variantcontext.writer.Options
+import htsjdk.variant.variantcontext.writer.VariantContextWriterBuilder
+import htsjdk.variant.vcf.VCFFileReader
+import htsjdk.variant.vcf.VCFHeader
+import htsjdk.variant.vcf.VCFHeaderLine
+import net.maizegenetics.phgv2.utils.Position
+import net.maizegenetics.phgv2.utils.parseALTHeader
+import org.apache.logging.log4j.LogManager
+import java.io.File
+import kotlin.math.abs
+
+/**
+ * this command takes a pathing hvcf file and a vcf file created by aligning reads to a composite genome
+ * (the latter from a program e.g. DeepVariant or Octopus) and creates a
+ * vcf file in haplotype coordinates.
+ * The incoming variantVCF file is in composite genome coordinates.
+ * The incoming path hvcf file is in reference coordinates.
+ * The output file will be in haplotype coordinates, calculated from the Regions information
+ * in the path hvcf file's ALT header information, combined with the reference coordinates from the path hvcf entries.
+ *
+ *
+ */
+data class HaplotypeData(val id: String, val refContig: String, val refStart: Int, val hapLen: Int)
+
+class CreateHaplotypeVCF: CliktCommand(help = "Create g.vcf file for a PHG pathing h.vcf using data from existing PHG created g.vcf files")  {
```

Comment:
I wonder if this needs to be named something else.  At first glance it seems that it just makes a hvcf but doesn't really say for what.  Maybe CreateRareAlleleVCF or CreateReseqeunceVCF?

Reply:
Maybe something about coordinates, like CreateHapCoordVCF? Still a little long, but it gets across that the coordinates are being manipulated.

Reply:
As per lab meeting today, Ed says to pick something and we can change later.  I have picked composite-to-haplotype-coords based on Cinta, Bethany and others comments.  I do not expect everyone to like this.  Will let Brandon make changes when/if he updates documentation.